### PR TITLE
Diverse and migrations

### PR DESCRIPTION
**Pull request by karteufel**
generally 'diverse' commit looks ok and if you won't find any business logic changes I'd recommend to roll back my changes. I like divided version more.

About "UPD: let's not merge back in one file" in task: I think we agreed verbally to do it (otherwise there's nothing to merge about, except just clicking  'merge' button

### CODE REVIEW

File: contracts/AccountModifiers.sol [hunk 1]
```diff
@@ -1,5 +1,7 @@
 pragma solidity ^0.4.13;
 
+pragma solidity ^0.4.13;
```
Comment (tokenstoredev) [inline contracts/AccountModifiers.sol]: Please remove

File: contracts/TokenStore.sol [hunk 2]
```diff
@@ -51,9 +51,10 @@ contract Ownable {
     _;
   }
 
-  function transferOwnership(address _newOwner) onlyOwner {
```
Comment (tokenstoredev) [inline contracts/TokenStore.sol]: Please revert that part, the old one is better.

Comment (karteufel) [reply contracts/TokenStore.sol]: @tokenstoredev 

There's tow 'old' versions: 

1) https://github.com/tokenstore/contract/commit/e61c9dd91db9ce8785f109cf0206b7e2565f4284#diff-7953b479a014e7309c8088e71766cf16L54

2) https://github.com/tokenstore/contract/commit/e61c9dd91db9ce8785f109cf0206b7e2565f4284#diff-8fff83b4119641a960ebf126d3729954L15

Which one we use in both cases? Or, should we leave 2 different versions in 2 different files as it was before?

File: contracts/AccountModifiers.sol (hunk 1 above)
Comment (karteufel) [reply contracts/AccountModifiers.sol]: @tokenstoredev removed extra line

**Comment by karteufel** [comment]
@tokenstoredev done with changes

**Pull request merged**